sonar-project.properties

```
@@ -0,0 +1,12 @@
+sonar.projectKey=elastic_integrations_AYu5LCaKQZlFqhqWIrk_
+sonar.host.url=https://sonar.elastic.dev
+sonar.language=flex
+sonar.log.level=DEBUG
+
+sonar.exclusions=**/**
+#TODO remove the line below after tests
+sonar.inclusions=.buildkite/scripts/**
+
+sonar.cobertura.reportPath=build/test-coverage/coverage-*.xml
```

It looks to me like the problem is the reportPath is misconfigured. You correctly make an effort to merge the XML reports into a single report which SonarQube can ingest, but I think you forgot to update [the path](https://github.com/elastic/integrations/pull/8464/files#diff-d03315e8c57a21b06ad57f62bbd018660a575b2877e78b285bd6910b924cd834R6) to `coverage_merged.xml`. 

```suggestion
sonar.cobertura.reportPath=build/test-coverage/coverage_merged.xml
```

---

Thank you @rhass for the involved in this.
I did it,
The result is the same.
Let's check it once again.